### ISSUE TITLE
Problem with NPM Install?

### ISSUE THREAD

**Issue by SteveIreland** [opensourcepos/opensourcepos#1699]
I'm trying to build a test environment for the current state of the master.

I downloaded the zip file, unzipped it and ran the usual `composer install` and then ran the `npm install`.  When running the `npm install` I encountered the following error.  Is this a  problem in the npm script or is this a problem in npm itself?

```

npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/130
npm WARN deprecated graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm ERR! path C:\Users\steve\Downloads\openSourcePos\jekkos-master-2a\opensourcepos-master\node_modules\acorn
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename 'C:\Users\steve\Downloads\openSourcePos\jekkos-master-2a\opensourcepos-master\node_modules\acorn' -> 'C:\Users\steve\Downloads\openSourcePos\jekkos-master-2a\opensourcepos-master\node_modules\.acorn.DELETE'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

**Comment by daN4cat** [comment]
Search Stackoverflow for any help. I don't have Windows installation anymore as I use Ubuntu 16.04, so I'm afraid I can't help.

**Comment by SteveIreland** [comment]
My Windows OS automatically upgraded itself about 3 or 4 days ago.  It took some time so it seemed like a significant one.  When I ran NPM yesterday it simply crashed and it was definitely due to the Windows upgrade. So I upgraded NPM and it behaved a lot better but then this error occurred.  Since I had to upgrade from a version 7 of NPM to a version 8 of NPM I was hoping that someone else was sharing my pain.  But if not I'll work it out.

**Comment by daN4cat** [comment]
Sorry to be helpless this time...

**Comment by SteveIreland** [comment]
Not that anyone here cares, but just in case someone else gets caught up in a similar issue I wanted to report that I was finally successful in creating a test environment.  I had to back out the Windows 1709 upgrade.   It not only broke composer and npm for me but I finally realized that it broke all tools that download and unzip files.  I spent hours searching for "quick fixes" but finally gave up and backed it out
